README.md

```diff
@@ -136,14 +136,18 @@ if err := stats.DeleteMeasure(m); err != nil {
 
 ### Creating an aggregation
 
-Currently only 2 types of aggregations are supported. The CountAggregation is used to count
+Currently 4 types of aggregations are supported. The CountAggregation is used to count
```

Comment:
Can you update the source here at https://github.com/census-instrumentation/opencensus-go/blob/master/internal/readme/source.md?

README is autogenerated by running the script, see https://github.com/census-instrumentation/opencensus-go/tree/master/internal/readme.

Sorry, this should be more obvious but I didn't have time yet to make it more visible.

Reply:
No problem, updated.